src/Validation/validateInput.js

```diff
@@ -0,0 +1,10 @@
+export default function validateInput(inputMail, inputPassword) {
+  var mailformat = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
+  if (inputMail == '' || mailformat.test(inputMail) == false) {
```

Comment:
Also, please use "===" instead of "==". You can explore the difference between both.